src/lib.rs

```diff
@@ -57,6 +57,8 @@ pub struct ThirdPersonCamera {
     pub focus: Vec3,
     pub gamepad_settings: CustomGamepadSettings,
     pub mouse_sensitivity: f32,
+    pub mouse_orbit_on_button_enabled: bool,
```

Comment:
Can you rename this to `mouse_orbit_button_enabled`

Reply:
done